Makefile

```diff
@@ -9,6 +9,13 @@ CLIPPY_OPTS := -D warnings -D clippy::clone_on_ref_ptr -D clippy::enum_glob_use
 test: ## Run all tests.
 	cargo test ${VERBOSE} --all -- --nocapture
 
+cov: ## Run code coverage.
+	# Tarpaulin only supports x86_64 processors running Linux.
+	# https://github.com/xd009642/tarpaulin/issues/161
+	export PATH="$(pwd)/devtools/cov:$PATH"
```

Comment:
This does not work for Makefile. In Makefile, every line is executed in its on sub process.

Also I'm curious whey the `$` works here, Makefile also use it for its variables and functions, they should be escaped in Makefile.